x-pack/solutions/security/plugins/security_solution/server/endpoint/routes/actions/details.ts

```diff
@@ -47,7 +47,7 @@ export const registerActionDetailsRoutes = (
         },
       },
       withEndpointAuthz(
-        { all: ['canReadSecuritySolution'] },
+        { all: ['canAccessEndpointActionsLogManagement'] },
```

Comment:
Just checking if this and all the above changes regarding privileges change are intentional, because it was not mentioned in the PR description :)